Riot/Modules/Room/CellData/RoomBubbleCellData.m

```diff
@@ -289,26 +289,28 @@ - (BOOL)hasNoDisplay
             hasNoDisplay = NO;
             break;
         case RoomBubbleCellDataTagPoll:
-            if (self.events.lastObject.isEditEvent) {
-                hasNoDisplay = YES;
+            if (!self.events.lastObject.isEditEvent) {
+                hasNoDisplay = NO;
             }
             
-            hasNoDisplay = NO;
             break;
         case RoomBubbleCellDataTagLocation:
             hasNoDisplay = NO;
             break;
         case RoomBubbleCellDataTagLiveLocation:
             // If the summary does not exist don't show the cell
-            if (!self.beaconInfoSummary)
+            if (self.beaconInfoSummary)
             {
-                hasNoDisplay = YES;
+                hasNoDisplay = NO;
             }
             
-            hasNoDisplay = NO;
             break;
         case RoomBubbleCellDataTagVoiceBroadcast:
-            hasNoDisplay = YES;
+            if (RiotSettings.shared.enableVoiceBroadcast == YES &&
+                [[VoiceBroadcastInfo modelFromJSON:self.events.lastObject.content].state isEqualToString:@"started"]) {
```

Comment:
You should not have a string comparison here.
If you change the `started` value from the enum, it won't work anymore.